source/openapi-admin-v3.yaml

```diff
@@ -850,45 +869,46 @@ paths:
                     - $ref: "#/components/schemas/AtlasCluster"
                     - $ref: "#/components/schemas/AtlasFederatedInstance"
                     - $ref: "#/components/schemas/ThirdPartyService"
-    post:
+    patch:
```

Comment:
We lost the real `adminCreateService` endpoint somewhere here - there should be an endpoint for `POST /groups/{groupId}/apps/{appId}/services/config`.

Let's re-add that endpoint and then update this one to be `adminUpdateServiceConfig` or similar with an appropriate `summary`.

Reply:
gah! Moving things around so much has led to quite the mess.